sbom/main.ts

```diff
@@ -16,7 +16,7 @@ interface Package {
     version: string;
     license: string;
     type: string;
-    dependencies: Dependency[];
+    dependencies: { name: String, version: String }[];
```

Comment:
I think https://www.typescriptlang.org/docs/handbook/2/objects.html#index-signatures is what we need here since we don't know the names of the dependency packages ahead of time.